Makefile

```diff
@@ -13,7 +13,7 @@ go-install:
 GOOS=$(shell go env GOOS)
 GOARCH=$(shell go env GOARCH)
 GOPATH=$(shell go env GOPATH)
-VERSION=0.9.0
+VERSION=0.10.0
```

Comment:
I find this a little bit confusing, should we rename this into something that'll indicate that it's the "edge" version?